README.md

```diff
@@ -64,10 +64,10 @@ app.setErrorHandler((err, req, reply) => {
 })
 ```
 
-If you need to handle POST routes, you can define the HTTP method:
+If you need to handle HEAD routes, you can define the HTTP method:
 ```js
 fastify.next('/api/*', { method: 'GET' });
-fastify.next('/api/*', { method: 'POST' });
+fastify.next('/api/*', { method: 'HEAD' });
```

Comment:
Thanks for the feedback. I tried to explain that in the description: Next doesn't support HTTP methods other than GET and HEAD since 12.0.16-canary.16. There's a link to the specific change in the description.

Reply:
ah ok, thanks